src/peft/peft_model.py

```diff
@@ -1408,6 +1408,26 @@ def base_model_torch_dtype(self):
     def active_peft_config(self):
         return self.peft_config[self.active_adapter]
 
+    def _get_peft_specific_model_tags(self):
+        """Derive tags for the model card from the adapter's config. For example, setting the
+        base model is important for enabling support for HF inference providers but it also makes models more
+        searchable on the HF hub.
+        """
+        peft_method = self.active_peft_config.peft_type.value
+
+        tags = []
+
+        if hasattr(self.base_model, "model") and isinstance(self.base_model.model, transformers.PreTrainedModel):
+            tags.append("transformers")
+
+        if peft_method == "LORA":
```

Comment:
Should we do it for all the other methods too i.e. qlora etc

Reply:
As discussed in slack, we're going to skip extending this feature for now until we know what tags we need.